openstack/sharedfilesystems/v2/messages/results.go

```diff
@@ -0,0 +1,101 @@
+package messages
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+const MinimumManilaMessagesMicroVersion = "2.37"
```

Comment:
microversions are set during the request, so if we would be adding constants like this, it would go in `requests.go` instead. However, let's remove this - this opens the door to having Gophercloud maintain these kinds of contants for every microversion version and feature.